index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"> 
+    <title>Personal Portfolio Website</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+
+        </header>
+        <header class="menu">
+            <div class="menu-logo">
+                <a>Welcome</a>
+            </div>
+            <nav class="menu-button">
+                <ul>
+                    <li class="bar"> </li>
+                    <li class="bar"> </li>
+                    <li class="bar"> </li>
+                </ul>
+            </nav>
+        </header>
+        <nav class="nav-menu">
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+        <section class="hero">
+            <section class="headline">
+                <div class="place primary">
+                    <h2> Hey there. I’m Alvin <br><span class="green"> I’m a software developer </span></h2>
+                </div>
+                <div class="place support">
+                    <p>
+                        I can help you build a product ,
+                        feature or website Look through
+                        some of my work and experience! 
+                        If you like what you see and have 
+                        a project you need coded, don’t 
+                        hesitate to contact me.
+                    </p>
+                </div>
+                <div class="social">
+                    <ul>
+                        <li ><a href=""><i class="fa fa-twitter"></i></a></li>
+                        <li ><a href=""><i class="fa fa-github"></i></a></li>
+                        <li ><a href=""><i class="fa fa-instagram"></i></a></li>
+                        <li ><a href=""><i class="fa fa-facebook"></i></a></li>
+                    </ul>
```

Comment:
Add the missing social icons
![right](https://user-images.githubusercontent.com/58376020/125834649-69577be6-0e07-4fa9-ba4d-4b68279a0ea6.png)

Reply:
This issue was well resolved ✔️